app/config/config.yml

```diff
@@ -208,6 +208,8 @@ scheb_two_factor:
 
     google:
         enabled: "%twofactor_auth%"
+        label: wallabag
```

Comment:
Travis does not seem happy with this change:

  https://travis-ci.org/github/wallabag/wallabag/jobs/667283180#L738

Reply:
It's called `server_name` in version 4.4 that we use.